data-prepper-plugins/rds-source/src/main/java/org/opensearch/dataprepper/plugins/source/rds/datatype/postgres/ColumnType.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ */
+
+package org.opensearch.dataprepper.plugins.source.rds.datatype.postgres;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public enum ColumnType {
+    BOOLEAN(16, "boolean"),
```

Comment:
Are those typeId from postgres? Is there way we can use their SDK variables instead of the hardcoded number

Reply:
Yes, they are from postgres, but I don't think they have those in their libraries. Postgres has a system table `pg_type` that contains those type information. So an alternative would be to query the `pg_type` table the type ids and other type information.